app/frontend/src/features/communities/constants.ts

```diff
@@ -16,12 +16,15 @@ export const COMMENT = "Comment";
 export const COMMENTS = "Comments";
 export const COMMUNITY_HEADING = (name: string) => `Welcome to ${name}!`;
 export const COMMUNITY_MODERATORS = "Community moderators";
+export const COMMUNITY_PAGE_UPDATED =
+  "The community page has been successfully updated.";
 export const COMMUNITY_TABS_A11Y_LABEL = "Tabs for community sub-pages";
 export const CREATED_AT = "Created at ";
 export const CREATE_NEW_DISCUSSION_TITLE = "Create new post";
 export const DISCUSSIONS_EMPTY_STATE = "No discussions at the moment.";
 export const DISCUSSIONS_LABEL = "Discussions";
 export const DISCUSSIONS_TITLE = "Discussions";
+export const EDIT_COMMUNITY_PAGE = "Edit community page";
```

Comment:
I wonder if this should be Edit local info? Can probably get away with changing this only and not all the components 😆